platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -621,18 +627,13 @@ public class SwirldsPlatform implements Platform {
         final FutureEventBuffer futureEventBuffer = new FutureEventBuffer(platformContext);
 
         // wire ISS output
-        final IssHandler issHandler =
-                new IssHandler(stateConfig, this::haltRequested, this::handleFatalError, issScratchpad);
-        final OutputWire<IssNotification> issOutput =
-                platformWiring.getIssDetectorWiring().issNotificationOutput();
-        issOutput.solderTo(
-                "issNotificationEngine", "ISS notification", n -> notificationEngine.dispatch(IssListener.class, n));
-        issOutput.solderTo("statusManager_submitCatastrophicFailure", "ISS notification", n -> {
-            if (Set.of(IssType.SELF_ISS, IssType.CATASTROPHIC_ISS).contains(n.getIssType())) {
-                platformStatusManager.submitStatusAction(new CatastrophicFailureAction());
-            }
-        });
-        issOutput.solderTo("issHandler", "ISS notification", issHandler::issObserved);
```

Comment:
why move these into the ISS handler?

Reply:
@lpetrovic05 These actions are part of how we handle an ISS: I think it's a lot more understandable and maintainable to have the responses to an ISS collected in a single spot. I don't think we gain anything from indirectly calling these methods via the wiring framework.

Reply:
@lpetrovic05 Bullet points 1 and 2 have been completed, so this is ready for a re-review